src/components/rightSidePanel/parameters/SidePanelSearch.vue

```diff
@@ -0,0 +1,79 @@
+<script setup lang="ts">
+import { refDebounced } from '@vueuse/core'
+import { ref, toRef, watch } from 'vue'
+
+import { cn } from '@/utils/tailwindUtil'
+
+const props = withDefaults(
+  defineProps<{
+    searcher?: (
+      query: string,
+      onCleanup: (cleanupFn: () => void) => void
+    ) => Promise<void>
+    updateKey?: (() => unknown) | unknown
+  }>(),
+  {
+    searcher: async () => {}
+  }
+)
+
+const searchQuery = defineModel<string>({ default: '' })
+
+const isQuerying = ref(false)
+const debouncedSearchQuery = refDebounced(searchQuery, 700, {
```

Comment:
```suggestion
const debouncedSearchQuery = refDebounced(searchQuery, 100, {
```
700 ms feels super unresponsive here. Can we lower it?